docs/contributing/gatsby-docs-translation-guide.md

```diff
@@ -0,0 +1,80 @@
+
+## Contributing to a translation
+
+Once a language repository is created and someone on the Gatsby team has assigned codeowners, contributions can begin. It is up to the discretion of the contributor how exactly they want to work, but it's recommended to limit the scope of PRs to 1 doc at a time to aid with code reviewing.
+
+Maintainers of a translation are expected to review translations for quality, friendly and encouraging tone of voice, and technical accuracy. These qualities should be considered in every pull request regardless of who it comes from; contributions should be encouraged from anyone in the community, not just maintainers.
+
+> ⚠️ Note: All contributors are expected to follow the [Gatsby Code of Conduct](/contributing/code-of-conduct/) and work professionally with fellow contributors. For issues with conduct, if you are unable to work things out amicably amongst yourselves (perhaps after filing a public issue or having a discussion on Discord), you can contact the Gatsby team at [conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com) with details about the situation.
+
+### Code review process
+
+Translation contributions must be opened as pull requests to provide time for review and comments. If actionable feedback is given on a PR review, the author must acknowledge the review and make changes with their discretion. Ignoring review feedback completely is not allowed (see response templates below for help with this).
+
+### Maintainer responsibilities
+
+As repo maintainers and members of the Gatsby community, your responsibilities are as follows:
+
+- Keep issues up to date as people volunteer to translate pages.
+- Review pull requests made by contributors promptly.
+- Review pull requests generated by gatsbybot in order to make sure translations remain up to date with the source repo.
+- Act as point of contact for your language and answer questions from both contributors to your language and the core Gatsby team.
+- Set up a process in order to get your translation published. (details to come)
+
+Additional tips:
+
+- Make a glossary and translation style guide. Here are some [examples from the React translation project](https://github.com/reactjs/reactjs.org-translation/blob/master/maintainer-guide.md#make-a-glossary-and-style-guide).
+- Collaborate on large or challenging pages. More than one person can contribute to a long tutorial or reference guide!
+- Set up a review process
+- Ask for help
+
+### Templates for responses on PRs
```

Comment:
what is this section supposed to be about?

Reply:
Does the intro paragraph on line 79 not explain it well enough?

Reply:
all cleared up on slack :)

Reply:
I'm updating it to "Template responses for closing PRs"